backoffice/workflows/api/views.py

```diff
@@ -70,6 +73,43 @@ def create(self, request, *args, **kwargs):
             return Response({"error": str(e)}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
 
 
+class AuthorWorkflowViewSet(viewsets.ViewSet):
+    def create(self, request):
+
+        workflow = Workflow.objects.create(data=request.data["data"], workflow_type=request.data["workflow_type"])
+        serializer = WorkflowSerializer(workflow)
+
+        if workflow.workflow_type == WORKFLOW_TYPES[2][0]:
+            dag_name = "author_create_initialization_dag"
+        elif workflow.workflow_type == WORKFLOW_TYPES[3][0]:
+            dag_name = "author_update_dag"
+        else:
+            return Response({"message": "workflow_type unrecognized"}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
```

Comment:
This should be `400`

Reply:
Also let's create an exception for invalid inputs https://www.django-rest-framework.org/api-guide/exceptions/#apiexception